contracts/staking/src/execute.rs

```diff
@@ -586,9 +586,10 @@ pub fn update_config(
     minimum_liquid_stake_amount: Option<Uint128>,
     multisig_address_config: Option<MultisigAddressConfig>,
     protocol_fee_config: Option<ProtocolFeeConfig>,
-    reserve_token: Option<String>,
+    native_token_denom: Option<String>,
```

Comment:
If i am understanding this correctly, the name `reserve_token` is now changed to `native_token_denom`? If so, there are still many places that use `reserve_token` (execute.rs, msg.rs, test codes, scripts)

Reply:
let's change them too.